test/integration/api/admin/api_spec.js

```diff
@@ -425,6 +427,51 @@ describe("api", function () {
         }));
       });
     });
+
+    describe('selective response', () => {
+      const expectedKeys = ['public_id', 'asset_id', 'folder', 'tags'].sort();
```

Comment:
Maybe add a comment here on why stating like this and sorting instead of defining in a sorted order upfront?